frontend/projects/marketplace/src/pages/show/additional/additional.component.html

```diff
@@ -31,41 +38,53 @@ <h2>Instructions</h2>
           </ion-item>
         </ion-item-group>
       </ion-col>
-      <ion-col sizeSm="12" sizeMd="6">
+      <ion-col sizeXs="12" sizeMd="6">
         <ion-item-group>
           <ion-item
-            [href]="pkg.manifest['upstream-repo']"
+            [href]="manifest['upstream-repo']"
             target="_blank"
             rel="noreferrer"
             detail="false"
           >
             <ion-label>
               <h2>Source Repository</h2>
-              <p>{{ pkg.manifest['upstream-repo'] }}</p>
+              <p>{{ manifest['upstream-repo'] }}</p>
             </ion-label>
             <ion-icon slot="end" name="open-outline"></ion-icon>
           </ion-item>
           <ion-item
-            [href]="pkg.manifest['wrapper-repo']"
+            [href]="manifest['wrapper-repo']"
             target="_blank"
             rel="noreferrer"
             detail="false"
           >
             <ion-label>
               <h2>Wrapper Repository</h2>
-              <p>{{ pkg.manifest['wrapper-repo'] }}</p>
+              <p>{{ manifest['wrapper-repo'] }}</p>
             </ion-label>
             <ion-icon slot="end" name="open-outline"></ion-icon>
           </ion-item>
           <ion-item
-            [href]="pkg.manifest['support-site']"
+            [href]="manifest['support-site']"
             target="_blank"
             rel="noreferrer"
             detail="false"
           >
             <ion-label>
               <h2>Support Site</h2>
-              <p>{{ pkg.manifest['support-site'] }}</p>
+              <p>{{ manifest['support-site'] }}</p>
```

Comment:
Support and marketing site are optional inclusions in the manifest - possibly do not display them if they do not exist.

Reply:
If either is missing, I think they should inherit the upstream repository URL. This would be done on the packing side or on the registry

Reply:
Or it could be done UI side, which would help for side loaded packages. Or we could just say something like "none provided" in the UI.

Reply:
Disagree that they should be inherited (meaning they shouldn't be auto populated with the upstream url by the UI if missing), as each field is intended for different purposes - and its really up to the service packager to populate them as seen fit. If they are left blank, the UI should account for such.

Agree with second point - having the item be disabled and display informative text.